app/src/main/java/org/stacks/app/data/interfaces/IIdentityRepository.kt

```diff
@@ -0,0 +1,25 @@
+package org.stacks.app.data.interfaces
+
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ApplicationComponent
+import kotlinx.coroutines.flow.Flow
+import org.stacks.app.data.IdentityModel
+import org.stacks.app.data.IdentityRepository
+
+interface IIdentityRepository {
+    suspend fun clear(): Boolean
+    suspend fun set(model: IdentityModel): Boolean
```

Comment:
Why booleans, it might not work? If it might not work, could it be a better result object?